src/docs/release/breaking-changes/add-currentAutofillScope-to-TextInputClient.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Add currentAutofillScope to TextInputClient
+description: A new getter`TextInputClient.currentAutofillScope` was added to the `TextInputClient` interface for autofill support.
+---
+
+## Summary
+
+A new getter`TextInputClient.currentAutofillScope` was added to the `TextInputClient` interface.
+
+## Context
+
+On many platforms, autofill services are capable of autofilling multiple input fields in a single autofill 
+attempt. For example, username fields and password fields can usually be autofilled in one go. For this reason, 
+a Flutter input field that is about to trigger autofill should also provide the platform with information about other autofillable
+input fields logically connected to it. `TextInputClient.currentAutofillScope` defines the group of input fields that are logically
+connected to this `TextInputClient`, and can be autofilled together.
+
+## Description of change
+
+`TextInputClient` now has an additional getter that returns the `AutofillScope` that this client belongs to. This getter will be used by the input client to collect autofill related information from other 
+autofillable input fields within the same scope.
+
```

Comment:
You'll need a `<!-- skip -->` tag here.  Whenever you have code that is not a complete example, the analyzer fails so Travis fails. I can't merge a PR if Travis fails.

Reply:
I expect the CI to pass once the proposed framework change lands. I'll add the tag if that doesn't fix this issue.

Reply:
On the website repo, Travis tests the code against the current stable release. So I do expect the code to still fail. (The current stable release is 1.12.)